web/src/features/auth/components/MoreCast2AuthWrapper.tsx

```diff
@@ -30,12 +31,20 @@ const MoreCast2AuthWrapper = ({ children }: Props) => {
         setRenderChildren(true)
       }
 
-      if (!window.location.href?.includes('access_token') && isAuthenticatedForecaster) {
+      const lastLoginString = localStorage.getItem('last_morecast_login') ?? '0'
+      const lastLogin = parseInt(lastLoginString)
+      const now = DateTime.now().toUnixInteger()
+
+      // Force a redirect to WF1 authentication if there is no access_token or the last time Morecast
+      // was logged into was more than 60 minutes ago in order to handle a user reloading a page with an
+      // old access token
+      if (!window.location.href?.includes('access_token') || now - lastLogin > 30) {
```

Comment:
Currently have the check set to 30 seconds for testing purposes. Will change to 3600 seconds before deployment and merge.